src/style.css

```
@@ -1 +1,80 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --link-color: #00acdc;
+}
+
+*,
+*:before,
+*:after {
```

```suggestion
* {
```
In my case that will be better?